auth/user_mgt.go

```diff
@@ -452,22 +571,21 @@ func handleServerError(err error) error {
 
 // Validators.
 
-func validateDisplayName(val interface{}) error {
-	if len(val.(string)) == 0 {
+func validateDisplayName(val string) error {
+	if len(val) == 0 {
 		return fmt.Errorf("display name must be a non-empty string")
 	}
 	return nil
 }
 
-func validatePhotoURL(val interface{}) error {
-	if len(val.(string)) == 0 {
+func validatePhotoURL(val string) error {
+	if len(val) == 0 {
 		return fmt.Errorf("photo url must be a non-empty string")
 	}
 	return nil
 }
 
-func validateEmail(val interface{}) error {
-	email := val.(string)
+func validateEmail(email string) error {
 	if email == "" {
```

Comment:
It's just slightly strange that you use `len(s) == 0` above but use `s == ""` here. I'm not sure which one would be preferred, but it would be nice to be consistent. Also, the actual error string is worded slightly differently.

Reply:
Changed to `== ""`

Reply:
It's just slightly odd that "photo url must be a non-empty string", but "email must not be empty". Minor nit, though.